src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -567,8 +569,8 @@ private void trigger(@NotNull final Button button)
             index = Integer.valueOf(((Label) pane).getLabelText());
         }
         final boolean trigger = LanguageHandler.format(ON).equals(button.getLabel());
-        final Permissions.Action action = Permissions.Action.values()[index];
-        final Permissions.Rank rank = Permissions.Rank.valueOf(actionsList.getParent().getID().toUpperCase());
+        final Action action = Action.values()[index];
+        final Rank rank = Rank.valueOf(actionsList.getParent().getID().toUpperCase());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)